src/css/states/states.less

```diff
@@ -163,10 +169,10 @@ body .jwplayer.jw-state-error {
 
 // Show preview
 
-.jw-state-idle,
+.jw-state-idle:not(.jw-flag-audio-player),
 .jwplayer.jw-state-complete:not(.jw-flag-casting):not(.jw-flag-audio-player),
-body .jw-error,
-body .jwplayer.jw-state-error {
+body .jw-error:not(.jw-flag-audio-player),
+body .jwplayer.jw-state-error:not(.jw-flag-audio-player) {
   .jw-preview {
     display: block;
   }
```

Comment:
Probably a better (and obvious) way of doing this, but couldn't figure it out. Only show preview for non controlbar only players